opentech/apply/funds/models/applications.py

```diff
@@ -108,7 +109,7 @@ def serve(self, request):
 
     content_panels = WorkflowStreamForm.content_panels + [
         FieldPanel('approval_form'),
-        FieldPanel('reviewers'),
+        FieldPanel('reviewers', widget=forms.SelectMultiple(attrs={'size': '16'})),
```

Comment:
@Parbhat A better way to do this?

Reply:
@frjo This is the best way as mentioned in [docs](http://docs.wagtail.io/en/v2.6.3/reference/pages/panels.html#wagtail.admin.edit_handlers.FieldPanel.widget).